src/__stories__/poster-card-story.tsx

```diff
@@ -64,6 +66,7 @@ export const Default: StoryComponent<PosterCardArgs> = ({
     title,
     subtitle,
     description,
+    withExtra,
```

Comment:
also we have some "icon" instead of "withIcon" (in tag story for example)

perhaps it is better to create a task to solve this "purity details" across all stories.

Reply:
I've created an issue for this: https://jira.tid.es/browse/WEB-1891